src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+public class Task {
+    protected String description;
+    protected int num;
+    protected boolean isDone;
```

Comment:
I like the naming of your boolean variables indicating that they are booleans using "is".